tutorials-and-examples/inference-servers/jetstream/maxtext/single-host-inference/terraform/maxengine/main.tf

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "kubernetes_deployment" "deployment_maxengine_server" {
+  metadata {
+    name = "maxengine-server"
+  }
+  spec {
+    replicas = 2
+    selector {
+      match_labels = {
+        app = "maxengine-server"
+      }
+    }
+    template {
+      metadata {
+        labels = {
+          app = "maxengine-server"
+        }
+      }
+      spec {
+        container {
+          args = [
+            "model_name=gemma-7b",
```

Comment:
OOC, do these parameters have to be hardcoded or could we convert these to variables for customers to adjust the model they use?